todo/src/index.js

```diff
@@ -1,443 +1,446 @@
 import React from "react";
 import { render } from "react-dom";
-import { StitchClientFactory } from "mongodb-stitch";
-import { browserHistory, Route } from "react-router";
+import { Route } from "react-router";
 import { BrowserRouter, Link } from "react-router-dom";
+import {
+Stitch,
+RemoteMongoClient,
+GoogleRedirectCredential,
+FacebookRedirectCredential
+} from "mongodb-stitch-browser-sdk";
 
 require("../static/todo.scss");
 
 let appId = "<your-app-id>";
+
 if (process.env.APP_ID) {
-  appId = process.env.APP_ID;
-}
+   appId = process.env.APP_ID;
+}  
 
 let mongodbService = "mongodb-atlas";
-if (process.env.MONGODB_SERVICE) {
-  mongodbService = process.env.MONGODB_SERVICE;
+   if (process.env.MONGODB_SERVICE) {
+   mongodbService = process.env.MONGODB_SERVICE;
 }
 
 let options = {};
-if (process.env.STITCH_URL) {
-  options.baseUrl = process.env.STITCH_URL;
+   if (process.env.STITCH_URL) {
```

Comment:
I know you guys might use this for internal testing but I think we shouldn't include it in any examples. There's currently no way for a user to talk to a Stitch other than the one hosting at `stitch.mongodb.com`